website/static/worker.js

```diff
@@ -2,65 +2,73 @@
 /* eslint no-var: off, strict: off */
 /* globals prettier prettierPlugins */
 
+var imported = {};
+function importScriptOnce(url) {
+  if (!imported[url]) {
+    imported[url] = true;
```

Comment:
Perhaps use an array, `Object.create(null)`, or a set here to avoid bugs we’ve encountered with using an object as a map on the past?

Reply:
I'll change it to use `Object.create(null)`, though I think it's almost impossible to encounter the same issue since we probably won't have some parser named something like `hasOwnProperty`.